cmd/utils/flags.go

```diff
@@ -780,11 +780,6 @@ var (
 		Usage: "Each enode URL in a pair is separated by a semicolon. Enode URL pairs are separated by a space. The format should be \"<proxy 0 internal facing enode URL>;<proxy 0 external facing enode URL>,<proxy 1 internal facing enode URL>;<proxy 1 external facing enode URL>,...\" (deprecated, use --proxy.proxyenodeurlpairs)",
 	}
 
-	ProxyCeloStatsFlag = cli.StringFlag{
```

Comment:
We should probably note that this flag is deprecated and doesn't do anything rather than immediately removing it as passing in undefined flags can cause problems.

Reply:
Yes, I thought about that, but my concern is that the flag only lives in the multi-proxy branch. So technically was not released to everyone.
I don't want to add something that will start as deprecated. @kevjue what do you think about this?

Reply:
If it was just from multi proxy, I definitely agree that we should remove it.